lmdeploy/pytorch/backends/cuda/graph_runner.py

```diff
@@ -134,9 +124,8 @@ def forward(self, **kwargs):
         context = self.ctx_mgr.current_context()
         self.model.update_context_cudagraph(self.meta, context)
         self._graph.replay()
-
         output_buffers = self.meta.output_buffers
-        output = self.slice_output(output_buffers, kwargs)
+        output = self.model.get_outputs_cudagraph(output_buffers, **kwargs)
```

Comment:
Should we update forward when  enable_graph=False so prefill and decoding can share same output format

Reply:
Done in line 213